tests/tests/test-stake.ts

```diff
@@ -19,6 +19,25 @@ describeWithMoonbeam("Moonbeam RPC (Stake)", `simple-specs.json`, (context) => {
 
   it("candidates set in genesis", async function () {
     const candidates = await context.polkadotApi.query.stake.candidates(GENESIS_ACCOUNT);
-    expect((candidates.toHuman() as any).id.toLowerCase()).equal(GENESIS_ACCOUNT);
+    expect(candidates.toHuman()["id"].toLowerCase()).equal(GENESIS_ACCOUNT);
+    expect(candidates.toHuman()["state"]).equal("Active");
+  });
+
+  it("inflation set in genesis", async function () {
+    const inflationInfo = await context.polkadotApi.query.stake.inflationConfig();
+    // {
+    //   expect: {
+    //     min: '100.0000 kUnit',
+    //     ideal: '200.0000 kUnit',
+    //     max: '500.0000 kUnit'
+    //   },
+    //   round: { min: '0.00%', ideal: '0.00%', max: '0.00%' }
+    // }
+    expect(inflationInfo.toHuman()["expect"]["min"]).to.eq("100.0000 kUnit");
+    expect(inflationInfo.toHuman()["expect"]["ideal"]).to.eq("200.0000 kUnit");
+    expect(inflationInfo.toHuman()["expect"]["max"]).to.eq("500.0000 kUnit");
+    expect(inflationInfo.toHuman()["round"]["min"]).to.eq("0.00%");
```

Comment:
@4meta5 that doesn't seem right.
Could it be that the --dev is not setting those correctly ?

Reply:
This is because this TS percent type is to the nearest hundredth place and the `Perbill` has 9 trailing zeros (parts per billion). So the per-round inflation is indeed less than `0.01%` and closer to `0.00%` so that's why this works.

Reply:
I added the values.